includes/licensing.php

```diff
@@ -999,20 +991,19 @@ public function get_version_information( $force_refresh = false ) {
         }
 
         $updates = array();
-
         foreach ( $this->items as $item ) {
             $item_key = $item['item_type'] . '-' . $item['id'];
 
             foreach ( $body as $item_information ) {
-				if ( $item_information->name !== $item['name'] ) {
+				if ( $item_information->slug !== $item['id'] ) {
```

Comment:
I'm looking into this a bit more, and it looks like this prevents modules from getting updates since the id and slug don't match. Here are a few examples:
slug: migrator, id: business-directory-migrate
slug: enhanced-categories, id: business-directory-categories
slug: elegant-grid-theme, id: elegant-grid

Is this tested for each module and theme?

Reply:
Maybe we should try testing this one?
```
if ( $item_information->name !== $item['name'] || empty( $item_information->license ) ) {
continue;
}
```

Reply:
Can confirm this is blocking my site from receiving the ratings & business card theme updates.

Reply:
Did some tests with the master branch and I also do not get plugin update notices. Was there an API change that could've affected this?

Reply:
No, there wasn't an api update for this. It just depends on the product. Some match the slug and some don't.

Reply:
Did more tests, I changed it to :

```
if ( trim( $item_information->name ) !== trim( $item['name'] ) || empty( $item_information->license ) ) {
      continue;
}
```

For update checks, these have to manually be triggered at the updates screen:

![image](https://user-images.githubusercontent.com/121492/150556360-e62fe27d-207f-4a8f-9997-73058349f9b5.png)

I did not see issues with other activated modules being deactivated also